src/TinyIoC/TinyIoC.cs

```diff
@@ -3582,7 +3582,7 @@ where localType.IsAssignableFrom(implementationType)
     }
     
     // TODO - find a better way to remove "system" assemblies from the auto registration
-        private readonly List<Func<Assembly, bool>> ignoredAssemlies = new List<Func<Assembly, bool>>()
+        private static readonly IReadOnlyList<Func<Assembly, bool>> ignoredAssemlies = new List<Func<Assembly, bool>>()

```

Comment:
I'm not sure how I feel about this being static. Would someone ever have more than one container instance?

Reply:
I added `IReadOnlyList` to avoid modifications of the list but then realized my IDE suggests static ;-)
The assembly ignore list will not change if you have multiple containers. It could be written as (static) if's inside `IsIgnoredAssembly` but the list seems to be more elegant. Since the list is identically for all containers and will never change it can be made static to reduce memory consumption.